queries/c/locals.scm

```diff
@@ -0,0 +1,34 @@
+;; Functions definitions
+(function_declarator
+  declarator: (identifier) @definition.function) @scope
+(preproc_function_def
+  name: (identifier) @definition.macro) @scope
+
+(preproc_def
+  name: (identifier) @definition.macro)
+(pointer_declarator
+  declarator: (identifier) @definition.variable)
+(parameter_declaration
+  declarator: (identifier) @definition.variable)
+(init_declarator
+  declarator: (identifier) @definition.variable)
+(array_declarator
+  declarator: (identifier) @definition.variable)
+(declaration
+  declarator: (identifier) @definition.variable)
+(enum_specifier
+  name: (*) @definition.type
+  (enumerator_list
+    (enumerator name: (identifier) @definition.variable)))
+
+;; Type / Struct / Enum
+(field_declaration
+  declarator: (field_identifier) @definition.field)
+(type_definition
+  declarator: (type_identifier) @definition.type)
+
+;; References
+(identifier) @reference
+
+;; Scope
+(for_statement) @scope
```

Comment:
we'll add more scopes another time i guess.
You should also update the README though.

Reply:
Actually, C is quite poor in scopes, even `for_statement` is added for convenience, the only real scope used in C is function based I think.

Reply:
well to me every condition blocks, while loops, do for loops, switch statement and so on are scopes. But that's ok we'll leave it for another time as i said :)

Reply:
That's the weird part of C, nothing despite functions and files are scopes ;-)

Reply:
well i'm not sure but if you do something like (that you shouldnt do though)
```c
if (something) {
  int a = 0;
...
}
```
the a is stored in a register and removed from the register when you are going out of the `if` scope ?

Reply:
You are right, how dumb am I. I'm adding the scopes right now.